sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMiningAsync.cs

```diff
@@ -33,33 +34,33 @@ public async Task AnalyzeSentimentWithOpinionMiningAsync()
                 "We changed rooms as the toilet smelled."
             };
 
-            AnalyzeSentimentResultCollection reviews = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { AdditionalSentimentAnalyses = AdditionalSentimentAnalyses.OpinionMining });
+            AnalyzeSentimentResultCollection analyzedReviews = await client.AnalyzeSentimentBatchAsync(reviews, options: new AnalyzeSentimentOptions() { MineOpinions = true });
 
-            Dictionary<string, int> complaints = GetComplaint(reviews);
+            Dictionary<string, int> complaintTargets = GetNegativeOpinionTargets(analyzedReviews);
```

Comment:
Let's call these targets instead of complaints, since complaint == opinion, which includes a target and a description.